trunk/SUAVE/Components/Energy/Converters/Motor.py

```diff
@@ -54,6 +55,7 @@ def __defaults__(self):
         self.gear_ratio         = 0.0
         self.gearbox_efficiency = 0.0
         self.expected_current   = 0.0
+        self.interpolated_func  = None
     
     def omega(self,conditions):
```

Comment:
Also missing docstring and doxygen info.

Reply:
Doxygen info added